cli/up/build.go

```diff
@@ -207,10 +436,10 @@ func getImage(images []types.ImageSummary, exp string) (id string, ok bool) {
 	return "", false
 }
 
-func getPushedImages(images []string, authToken string) map[image]struct{} {
+func getPushedImages(images []string, authToken string) map[string]struct{} {
```

Comment:
Any reason why you changed this to `map[string]struct{}` from `map[image]struct{}`?

Reply:
Before, in `buildImages`, we had the full image struct (including ID) to check against. Now when we're doing the check, we don't necessarily have the image ID, so we need to be able to query by name only (just string).

Reply:
Actually, I guess maybe we could. This may have just been something that changed when I had it structured some other way and I never noticed that I could change it back.